packages/ui/__tests__/Code.test.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { mount } from 'enzyme'
+
+import { Code } from '../src/Code'
+
+describe('Code', () => {
+  it('Renders the default Code correctly', () => {
+    const wrapper = mount(<Code value="Hello World" />)
+
+    // following does not work because of how CodeMirror creates itself
```

Comment:
Why does not it work? Could you be a bit more specific?

Reply:
while documenting it, i found a neat trick to resolve it :)) resolved.